src/Nethermind/Nethermind.JsonRpc.TraceStore/TraceSerializer.cs

```diff
@@ -0,0 +1,66 @@
+﻿//  Copyright (c) 2021 Demerzel Solutions Limited
+//  This file is part of the Nethermind library.
+//
+//  The Nethermind library is free software: you can redistribute it and/or modify
+//  it under the terms of the GNU Lesser General Public License as published by
+//  the Free Software Foundation, either version 3 of the License, or
+//  (at your option) any later version.
+//
+//  The Nethermind library is distributed in the hope that it will be useful,
+//  but WITHOUT ANY WARRANTY; without even the implied warranty of
+//  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+//  GNU Lesser General Public License for more details.
+//
+//  You should have received a copy of the GNU Lesser General Public License
+//  along with the Nethermind. If not, see <http://www.gnu.org/licenses/>.
+//
+
+using System.IO.Compression;
+using Nethermind.Core.Collections;
+using Nethermind.Evm.Tracing.ParityStyle;
+using Nethermind.JsonRpc.Modules.Trace;
+using Nethermind.Serialization.Json;
+using Newtonsoft.Json;
+
+namespace Nethermind.JsonRpc.TraceStore;
+
+public static class TraceSerializer
+{
+    private static readonly IJsonSerializer _jsonSerializer = new EthereumJsonSerializer();
+    private static readonly byte[] _emptyBytes = { 0 };
```

Comment:
There is a `0` here, is that like a RLP standard?

Reply:
Placeholder for empty value, I want to have a differentiation with just empty bytes.